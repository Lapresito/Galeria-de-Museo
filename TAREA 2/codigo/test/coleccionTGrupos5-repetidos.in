crearColeccionGrupos

visitanteMasRepetido
obtenerVisitantesRepetidos

crearGrupoABB
crearVisitante 2 Casimira Garcia 34
agregarAGrupoABB
crearVisitante 1 Artemio Rodriguez 21
agregarAGrupoABB
agregarAColeccionGrupos

crearGrupoABB
crearVisitante 2 Casimira Garcia 34
agregarAGrupoABB
crearVisitante 1 Artemio Rodriguez 21
agregarAGrupoABB
crearVisitante 3 Brunilda Gonzalez 20
agregarAGrupoABB
agregarAColeccionGrupos

visitanteMasRepetido
obtenerVisitantesRepetidos

crearGrupoABB
crearVisitante 2 Casimira Garcia 34
agregarAGrupoABB
agregarAColeccionGrupos

crearGrupoABB
crearVisitante 2 Casimira Garcia 34
agregarAGrupoABB
crearVisitante 4 Zenobia Ruiz 22
agregarAGrupoABB
agregarAColeccionGrupos

crearGrupoABB
crearVisitante 2 Casimira Garcia 34
agregarAGrupoABB
crearVisitante 4 Zenobia Ruiz 22
agregarAGrupoABB
agregarAColeccionGrupos

visitanteMasRepetido
obtenerVisitantesRepetidos

liberarColeccionGrupos
Fin
