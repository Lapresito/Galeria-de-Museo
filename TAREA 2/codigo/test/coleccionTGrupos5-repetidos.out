Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> La coleccion de grupos ha sido creado de forma exitosa.
2> No hay visitante repetido porque no hay visitantes en los grupos de la coleccion.
3> No hay visitantes repetidos entre los grupos.
4> El grupoABB ha sido creado de forma exitosa.
5> Visitante creadox de forma exitosa.
6> Se ha agregado el visitante al grupo de forma exitosa.
7> Visitante creadox de forma exitosa.
8> Se ha agregado el visitante al grupo de forma exitosa.
9> Se ha agregado el grupo a la coleccion de forma exitosa.
10> El grupoABB ha sido creado de forma exitosa.
11> Visitante creadox de forma exitosa.
12> Se ha agregado el visitante al grupo de forma exitosa.
13> Visitante creadox de forma exitosa.
14> Se ha agregado el visitante al grupo de forma exitosa.
15> Visitante creadox de forma exitosa.
16> Se ha agregado el visitante al grupo de forma exitosa.
17> Se ha agregado el grupo a la coleccion de forma exitosa.
18> El visitante que aparece en la mayor cantidad de grupos es:
Visitante Artemio Rodriguez
Id: 1
Edad: 21
19> Los visitantes repetidos son:
Visitante Artemio Rodriguez
Id: 1
Edad: 21
Visitante Casimira Garcia
Id: 2
Edad: 34
20> El grupoABB ha sido creado de forma exitosa.
21> Visitante creadox de forma exitosa.
22> Se ha agregado el visitante al grupo de forma exitosa.
23> Se ha agregado el grupo a la coleccion de forma exitosa.
24> El grupoABB ha sido creado de forma exitosa.
25> Visitante creadox de forma exitosa.
26> Se ha agregado el visitante al grupo de forma exitosa.
27> Visitante creadox de forma exitosa.
28> Se ha agregado el visitante al grupo de forma exitosa.
29> Se ha agregado el grupo a la coleccion de forma exitosa.
30> El grupoABB ha sido creado de forma exitosa.
31> Visitante creadox de forma exitosa.
32> Se ha agregado el visitante al grupo de forma exitosa.
33> Visitante creadox de forma exitosa.
34> Se ha agregado el visitante al grupo de forma exitosa.
35> Se ha agregado el grupo a la coleccion de forma exitosa.
36> El visitante que aparece en la mayor cantidad de grupos es:
Visitante Casimira Garcia
Id: 2
Edad: 34
37> Los visitantes repetidos son:
Visitante Artemio Rodriguez
Id: 1
Edad: 21
Visitante Casimira Garcia
Id: 2
Edad: 34
Visitante Zenobia Ruiz
Id: 4
Edad: 22
38> Coleccion de grupos liberada
39> Fin.
